.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishMergedExtractorConfiguration.cs

```diff
@@ -64,6 +64,38 @@ public class EnglishMergedExtractorConfiguration : BaseDateTimeOptionsConfigurat
 
             // (the)? (day|week|month|year)
             new Regex(DateTimeDefinitions.SingleAmbiguousTermsRegex, RegexFlags),
+
+        };
+
+        // Skip Regexes under TasksMode
+        public static readonly Regex[] TasksSkipMergeRegexes =
```

Comment:
Please surround creation/load with a check for the proper mode being set. If mode is not set, this should not be created.
Also, is the set of regexes in this array always the same for all languages? If so, we should declare it only in one place instead of all configs.

Reply:
For creation/load "TasksSkipMergeRegexes", I have followed similar convention as it's done under CalendarMode for creation for "TermFilterRegexes".

Reply:
TasksSkipMergeRegexes :  This can be different under different languages, right now we are not sure how this filter will work for other languages, so it's assigned as empty array

Reply:
> For creation/load "TasksSkipMergeRegexes", I have followed similar convention as it's done under CalendarMode for creation for "TermFilterRegexes".

Reply:
> Please surround creation/load with a check for the proper mode being set. If mode is not set, this should not be created. Also, is the set of regexes in this array always the same for all languages? If so, we should declare it only in one place instead of all configs.

For creation/load "TasksSkipMergeRegexes", I have followed similar convention as it's done under CalendarMode for creation for "TermFilterRegexes".

Reply:
> Please surround creation/load with a check for the proper mode being set. If mode is not set, this should not be created. Also, is the set of regexes in this array always the same for all languages? If so, we should declare it only in one place instead of all configs.

TasksSkipMergeRegexes :  This can be different under different languages, right now we are not sure how this filter will work for other languages, so it's assigned as empty array